test/test_backend.py

```diff
@@ -237,6 +239,20 @@ def test_subgraph(self):
         │   43530961 │ Induced termination of pregnancy complicated by …│ Condition │ SNOMED        │ Clinical Finding │ S                │ 609507007    │ ∅              │          1 │ condition_occurrence               │        20 │ 20                  │
         │            │…cardiac failure                                  │           │               │                  │                  │              │                │            │                                    │           │                     │
         └────────────┴──────────────────────────────────────────────────┴───────────┴───────────────┴──────────────────┴──────────────────┴──────────────┴────────────────┴────────────┴────────────────────────────────────┴───────────┴─────────────────────┘
+        
+"vocabulary_id_1"|"concept_id_1"|"concept_name_1"                                               |"concept_code"|"relationship_id"|"vocabulary_id_2"|"concept_id_2"|"concept_name_2"
```

Comment:
Big docstrings! But maybe fine temporary (or permanent) as I imagine it is helpful to describe the relationships in the test case!

Reply:
That was my doing. I wanted all that in there for going back and understanding what's going on, and because we're using this for testing Shahim's stuff.